### ISSUE TITLE
Project: npm audit shows security vulnerabilities

### ISSUE THREAD

**Issue by saadmhybris** [SAP/fundamental#591]
```
found 254 vulnerabilities (90 low, 99 moderate, 64 high, 1 critical) 
in 20686 scanned packages
 run `npm audit fix` to fix 149 of them.
 103 vulnerabilities require semver-major dependency updates.
 2 vulnerabilities require manual review. See the full report for details.
```

**Comment by saadmhybris** [comment]
this is being addressed under #478

**Comment by saadmhybris** [comment]
reopening as the issue was not fixed with #478
